mopidy_musicbox_webclient/static/js/controls.js

```diff
@@ -236,7 +236,7 @@ function removeTrack() {
     }
     var track = {};
     track.uri = [currentplaylist[i].uri];
-    mopidy.tracklist.remove(track);
+    mopidy.tracklist.remove({'uri':track.uri});
```

Comment:
It might have worked because of JS objects is simply "dicts", but the JSON deserializer on the server side is supposed to convert dicts with the `"__model__": "Track"` attribute back to a Python `Track` object before passing it to `remove()`.

Anyway, this is way more correct.